opennlp-tools/src/main/java/opennlp/tools/formats/brat/BratNameSampleStreamFactory.java

```diff
@@ -148,7 +154,15 @@ else if ("whitespace".equals(tokenizerName)) {
       }
     }
 
-    return new BratNameSampleStream(sentDetector, tokenizer, samples);
+    Set<String> nameTypes = null;
+    if (params.getNameTypes() != null) {
+      String[] nameTypesArr = params.getNameTypes().split(",");
+      if (nameTypesArr != null && nameTypesArr.length > 0) {
```

Comment:
String.split(",) would never return null,it would return an empty string if it is called like this ",".split(",")